e2e-tests/cypress/e2e/journeys/insurance/your-buyer/your-buyer.spec.js

```diff
@@ -0,0 +1,139 @@
+import {
+  headingCaption,
+  heading,
+  submitButton,
+  saveAndBackButton,
+} from '../../../pages/shared';
+import { insurance } from '../../../pages';
+import partials from '../../../partials';
+import {
+  BUTTONS,
+  LINKS,
+  ORGANISATION,
+  PAGES,
+} from '../../../../../content-strings';
+import { ROUTES } from '../../../../../constants';
+import { INSURANCE_ROOT } from '../../../../../constants/routes/insurance';
+import getReferenceNumber from '../../../helpers/get-reference-number';
+
+const { START } = ROUTES.INSURANCE;
+const insuranceStartRoute = START;
+
+const CONTENT_STRINGS = PAGES.INSURANCE.YOUR_BUYER_DETAILS;
+
+// const FIELD_ID = yourBuyerFiledVariables;
+
+const { taskList } = partials.insurancePartials;
+
+const goToPageDirectly = (referenceNumber) => {
+  cy.visit(`${INSURANCE_ROOT}/${referenceNumber}${ROUTES.INSURANCE.YOUR_BUYER.YOUR_BUYER_DETAILS}`, {
+    auth: {
+      username: Cypress.config('basicAuthKey'),
+      password: Cypress.config('basicAuthSecret'),
+    },
+  });
+};
+
+context('Insurance - Your Buyer - Type of yourbuyer Page - As an exporter, I want to enter the type of Your Buyer details I need for my export contract', () => {
+  let referenceNumber;
+
+  before(() => {
+    cy.visit(ROUTES.INSURANCE.START, {
+      auth: {
+        username: Cypress.config('basicAuthKey'),
+        password: Cypress.config('basicAuthSecret'),
+      },
+    });
+
+    cy.submitInsuranceEligibilityAndStartApplication();
+
+    taskList.prepareApplication.tasks.policyTypeAndExports.link().click();
```

Comment:
We can remove this (needs to be updated in a future PR to click into "your buyer" section).

Instead for now, we can remove this and use the `goToPageDirectly` function to get to the your buyer